sources/packages/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.controller.ts

```diff
@@ -207,4 +224,55 @@ export class ConfirmationOfEnrollmentController {
     // Send a message to allow the workflow to proceed.
     await this.workflow.sendConfirmCOEMessage(application.assessmentWorkflowId);
   }
+
+  /**
+   * Deny the Confirmation Of Enrollment(COE), defining the
+   * COE status as Declined in the student application table.
+   * @param locationId location that is completing the COE.
+   * @param applicationId application id to be updated.
+   * @param payload contains the denied reason of the
+   * student application.
+   */
+  @HasLocationAccess("locationId")
+  @Patch(
+    ":locationId/confirmation-of-enrollment/application/:applicationId/deny",
+  )
+  async denyConfirmationOfEnrollment(
+    @Param("locationId") locationId: number,
+    @Param("applicationId") applicationId: number,
+    @Body() payload: DenyConfirmationOfEnrollmentDto,
+  ): Promise<void> {
+    try {
+      await this.applicationService.setDeniedReasonForConfirmationOfEnrollment(
```

Comment:
The name "COE" and "ConfirmationOfEnrollment" both used randomly. We can use consistently or at least use COE when the function/property names become very lengthy.

Reply:
update lengthy function name with COE